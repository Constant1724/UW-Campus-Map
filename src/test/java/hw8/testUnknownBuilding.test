# Test Single Unknown Building
r
CSE
CS
# Test Double Unknown Buildings
r
PAP
cse
r
PPP
q
q