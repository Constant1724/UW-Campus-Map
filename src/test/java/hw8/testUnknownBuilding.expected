Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Test Single Unknown Building
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: CS

Enter an option ('m' to see the menu): # Test Double Unknown Buildings
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: PAP
Unknown building: cse

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: PPP
Unknown building: q

Enter an option ('m' to see the menu): 